catalog/src/test/java/org/killbill/billing/catalog/TestVersionedCatalog.java

```diff
@@ -44,18 +39,6 @@
     // WeaponsHireSmall-1.xml
     final DateTime dt1 = new DateTime("2010-01-01T00:00:00+00:00");
 
-    // WeaponsHireSmall-2.xml
```

Comment:
I saw these catalogs were copied over to `subscription`, but do we still need them here in `catalog`?

Reply:
I did not copy them -- i created new versions with just the plans we need inside subscriptions -- but for simplicity kept their same name. 

Those are still in use in `catalog` (e.g `TestVersionedCatalogLoader#testLoad` and also through `beatrix` (e.g `TestWithCatalogPlugin`).